packages/itwin/measure-tools/src/measurements/LocationMeasurement.ts

```diff
@@ -233,24 +234,27 @@ export class LocationMeasurement extends Measurement {
 
   private async createTextMarker(): Promise<void> {
     const adjustedLocation = this.adjustPointForGlobalOrigin(this._location);
+    let convertedSheetPoint;
+    if (this.drawingMetadata?.transform)
```

Comment:
Instead of copying this code over and over, consider introducing something similar to `adjustPointForGlobalOrigin`. We could also have both adjustments inside a single function.

Reply:
Adding to this, please use a more meaningful name for `this.drawingMetadata.transform`, like `drawingToWorld`